src/Actions/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,155 @@
+﻿Axe.Windows.Actions.CaptureAction
```

Comment:
Should these be in PublicAPI.Shipped.txt?

I _really_ hope this list is auto-generated...

Reply:
Files eliminated